addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -25,6 +25,13 @@ publicWidget.registry.ProductWishlist = publicWidget.Widget.extend(VariantMixin,
     init: function (parent) {
         this._super.apply(this, arguments);
         this.wishlistProductIDs = JSON.parse(sessionStorage.getItem('website_sale_wishlist_product_ids') || '[]');
+        if (!this.wishlistProductIDs.length) {
+            // Set a placeholder that will be recomputed on willStart method because
+            // the length of this.wishlistProductIDs will be the same that showed on
```

Comment:
I don't quite get this comment about the length being the same.

Reply:
‘wishlistProductIDs’ is a list containing the identifiers of the products that a user has added to their wishlist in Odoo. The counter reflects the length of the list. En este caso, la longitud de wishlistProductIDs debe coincidir con el número de productos que el usuario ha añadido a su lista de deseos. 
To prevent the on-screen counter from displaying an incorrect value while the actual data is being loaded, the on-screen counter value is used to create a temporary array of the same length. In this way, the length of wishlistProductIDs and the number displayed by the counter visually match, until the actual products are loaded and update the value of the list.

Maybe the comment is not too clear?